support/scripts/multiboot.py

```diff
@@ -180,6 +207,11 @@ def main():
         # Write equivalent ELF32 header
         elf32.write(elf32_ehdr)
 
+        # Append the Multiboot header
+        # NOTE: Multiboot header must be 4-byte aligned - luckily that is the
```

Comment:
You could add an assert for this.

Also I think there should be an assertion that you don't accidentally overwrite data past the original ELF header.

Reply:
> Also I think there should be an assertion that you don't accidentally overwrite data past the original ELF header.

It supposedly does not exist until I write it, so there is nothing to overwrite :thinking:.

Reply:
But you are opening the original input file. In the beginning, there is an ELF64 header there. And then you write a ELF32 header there without potentially moving the "data" of the file backwards.

Reply:
Sorry, I misunderstood the script :+1: